neuralpp/inference/graphical_model/representation/table/pytorch_table.py

```diff
@@ -215,6 +215,37 @@ def __mul__(self, other):
             array_of_potentials = self.potentials_tensor() * other.potentials_tensor()
             return self.from_array(array_of_potentials, result_is_batch)
 
+    def __add__(self, other):
```

Comment:
Wow, you really did put in a lot of work into this! But we don't need this new code! :-)